source/docs/software/hardware-apis/motors/wpi-drive-classes.rst

```diff
@@ -345,22 +340,22 @@ The MecanumDrive class contains two different default modes of driving your robo
     .. code-tab:: java
 
         public void teleopPeriodic() {
-            m_robotDrive.driveCartesian(-m_stick.getY(), m_stick.getX(), m_stick.getZ());
-            m_robotDrive.drivePolar(-m_stick.getY(), m_stick.getX(), m_stick.getZ());
+            m_robotDrive.driveCartesian(-m_stick.getY(), -m_stick.getX(), -m_stick.getZ());
+            m_robotDrive.drivePolar(-m_stick.getY(), -m_stick.getX(), -m_stick.getZ());
```

Comment:
This isn't really a correct usage of drivePolar anyway

Reply:
It's also not a valid function call- drivePolar requires a rotation2d. I'll try to fix this later.

Would converting the joystick "coordinates" to polar and using those be an acceptable demonstration, even though it's not a real world use case?

Reply:
Is there a real world use case of `drivePolar`?

Reply:
Polar might be easier to use in auto.

From usage reporting, 3 teams used drivePolar. However, a team that used polar in auto and cartesian in teleop I think would return cartesian in usage reporting.

Reply:
Are there any uses of drivePolar in the example projects?

Reply:
Reviewing uses of drivePolar across github, I didn't see any that used it from joysticks. Using it with hard coded angles seems to be how it's used